runtime/src/accounts_index.rs

```diff
@@ -263,18 +277,24 @@ impl<T: 'static + Clone + IsCached> WriteAccountMapEntry<T> {
         slot: Slot,
         account_info: T,
         reclaims: &mut SlotList<T>,
+        reclaims_must_be_empty: bool,
```

Comment:
hmmm based on the usage of this flag in this function, because:
1) We never check that `reclaims.is_empty()`, we just never add to the `reclaims` list
2) This really just checks the previous entry was cached

is it more explicit to rename `reclaims_must_be_empty` to `previous_slot_entry_was_cached`? 

We also never check that `reclaims.is_empty()`, we just never add to `reclaims`, so

Reply:
ok, renamed